test/e2e-tests.sh

```diff
@@ -45,9 +45,6 @@ initialize $@
 
 header "Running tests"
 
-# TODO: Real integration tests here instead of just verifying that we
-# can list Knative Services.
-./kn service get || fail_test
-echo "kn service get returned success"
+go_test_e2e ./test/e2e || fail_test
```

Comment:
I think we'll need to set path env var in this script context. From the tests failure, I think its not able to locate `kn` binary since it can't find it in path.

Reply:
Or you can just run `${REPO_ROOT_DIR}/kn` because that's where it's generated, from line 34. However, I agree that putting `kn` in the path is more flexible (specially if you want to `go test` directly from command line).

Reply:
OK made some changes. Works locally. Let's see if will also work on server.

Reply:
It did. :)